.vscode/tasks.json

```diff
@@ -0,0 +1,147 @@
+      "command": "pnpm run build",
+      "detail": "Build all"
+    },
+    {
+      "label": "Custom build",
+      "type": "shell",
+      "command": "pnpm run build --filter=${input:workspaceName}",
+      "detail": "Build specific package or application"
+    },
+    {
+      "label": "Run tests",
+      "type": "shell",
+      "command": "pnpm run test",
+      "detail": "Run all tests without filter"
+    },
+    {
+      "label": "Run tests for a specific component",
+      "type": "shell",
+      "command": "pnpm run test ${input:componentName}",
+      "detail": "Run all tests for a specific component"
+    },
+    {
+      "label": "Open development servers",
+      "type": "shell",
+      "command": "pnpm dev",
+      "detail": "Open all development servers"
+    },
+    {
+      "label": "Run tests with filter",
+      "type": "shell",
+      "command": "pnpm run test --filter=${input:testFilter}",
+      "detail": "Run different test types"
+    }
+  ],
+  "inputs": [
+    {
+      "type": "pickString",
+      "id": "workspaceName",
+      "description": "Choose a worskspace",
+      "options": [
+        "css-workshop",
+        "react-workshop",
+        "portal",
+        "@itwin/itwinui-css",
+        "@itwin/itwinui-react",
+        "@itwin/itwinui-variables"
+      ]
+    },
+    {
+      "type": "pickString",
+      "id": "componentName",
+      "description": "Choose a component",
+      "options": [
+        "Alert",
+        "Avatar",
+        "AvatarGroup",
+        "Backdrop",
+        "Badge",
+        "Breadcrumbs",
+        "ButtonGroup",
+        "Buttons",
+        "Carousel",
+        "Checkbox",
+        "ColorPicker",
+        "ComboBox",
+        "DatePicker",
+        "Dialog",
+        "Divider",
+        "DropdownMenu",
+        "ExpandableBlock",
+        "Fieldset",
+        "FileUpload",
+        "Flex",
+        "Footer",
+        "Header",
+        "Icon",
+        "InformationPanel",
+        "Input",
+        "InputGrid",
+        "InputGroup",
+        "InputWithDecorations",
+        "Label",
+        "LabeledInput",
+        "LabeledSelect",
+        "LabeledTextarea",
+        "LinkAction",
+        "List",
+        "Menu",
+        "Modal",
+        "NonIdealState",
+        "NotificationMarker",
+        "Overlay",
+        "Popover",
+        "ProgressIndicators",
+        "Radio",
+        "RadioTiles",
+        "SearchBox",
+        "Select",
+        "SideNavigation",
+        "SkipToContentLink",
+        "Slider",
+        "StatusMessage",
+        "Stepper",
+        "Surface",
+        "Table",
+        "Tabs",
+        "Tag",
+        "Textarea",
+        "ThemeProvider",
+        "Tile",
+        "TimePicker",
+        "Toast",
+        "ToggleSwitch",
+        "Tooltip",
+        "TransferList",
+        "Tree",
+        "Typography",
+        "VisuallyHidden"
+      ]
+    },
+    {
+      "type": "pickString",
+      "id": "testFilter",
+      "description": "Choose a test type",
```

Comment:
nit: This description can be improved to mention that it's asking to pick from a list of _workspaces_ to run tests for. ("test type" is too vague and unhelpful)

Reply:
I have made the change.